instrumentation/grpc-1.5/testing/src/main/groovy/io/opentelemetry/instrumentation/grpc/v1_5/AbstractGrpcTest.groovy

```diff
@@ -77,8 +85,6 @@ class GrpcTest extends AgentTestRunner {
             "${SemanticAttributes.RPC_SYSTEM.key()}" "grpc"
             "${SemanticAttributes.RPC_SERVICE.key()}" "example.Greeter"
             "${SemanticAttributes.RPC_METHOD.key()}" "SayHello"
-            "${SemanticAttributes.NET_PEER_NAME.key()}" "localhost"
```

Comment:
This is a regression, but I want to fix forward. Previously auto instrumentation got the address from the channel builder, but it seems weird to force that into library instrumentation too. gRPC does have a remote transport address attribute with some behavior difference between versions. Want to dig into that more as a followup instead of having to add address as a parameter to the interceptor's factories. How does that sound?

Reply:
No objection from me, let's create a tracking issue for this if @iNikem does not object.